### PR TITLE
C367964 Verify text of success toast when creating remote storage configurations (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by artsiomBandarenka**
[C367964](https://foliotest.testrail.io/index.php?/cases/view/367964)

[Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4535/allure/)

### CODE REVIEW

**Review by Scorpibear — CHANGES_REQUESTED** [review]
Please, split code of steps by empty lines/comments for better tracebility between manual and automated steps

**Review by Scorpibear — APPROVED** [review]

File: cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  });
+
+  it(
+    'C367964 Verify text of success toast when creating remote storage configurations (firebird) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+    () => {
+      // #1 Go to the "Settings" app
+      cy.visit(TopMenu.settingsPath);
+      // #2 Select "Remote storage" by clicking on it
+      SettingsMenu.openRemoteStorageSettings();
```
Comment (nayimovag) [inline cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js:33]: combine step 1 and 2 into cy.visit(SettingsMenu.remoteStoragePath)

File: cypress/support/fragments/settings/remote-storage/configurations.js [hunk 2]
```diff
@@ -48,35 +59,66 @@ function saveAndCloseForm() {
...
+      this.fillRequiredFields(name);
       saveAndCloseForm();
     },
+    fillRequiredFields(name, timing = '1') {
+      fillGeneralInfo(name, this.title);
+      cy.do(configurationFields.timingInput.fillIn(timing));
+    },
+    verifyRequiredFields(name, timing = '1') {
+      cy.expect(configurationFields.nameInput.has({ value: name }));
+      cy.expect(configurationFields.timingInput.has({ value: timing }));
```
Comment (nayimovag) [inline cypress/support/fragments/settings/remote-storage/configurations.js:88]: if multiple assertions in a row, better to pass as an array, like so: 
      cy.expect([
        configurationFields.nameInput.has({ value: name }),
        configurationFields.timingInput.has({ value: timing })
      ]);

File: cypress/support/fragments/settings/remote-storage/configurations.js [hunk 3]
```diff
@@ -48,35 +59,66 @@ function saveAndCloseForm() {
...
+      returningWorkflow = 'Items received at remote storage scanned into FOLIO',
+    ) {
+      expect(configurationFields.nameInput.has({ value: name }));
+      expect(configurationFields.provider.has({ value: including(this.title) }));
+      expect(
+        configurationFields.accessionHoldingWorkflowDropdown.has({
+          value: accessionHoldingWorkflow,
+        }),
+      );
+      expect(configurationFields.returningWorkflowDropdown.has({ value: returningWorkflow }));
```
Comment (nayimovag) [inline cypress/support/fragments/settings/remote-storage/configurations.js:121]: if multiple assertions in a row, better to pass as an array

File: cypress/support/fragments/settings/remote-storage/remoteStorage.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { NavListItem, Pane } from '../../../../../interactors';
+
+const remoteStorageNavigationPane = Pane('Remote storage');
+const configurationsItem = NavListItem('Configurations');
+const accessionTablesItem = NavListItem('Accession tables');
+
+export default {
+  checkSettingItems() {
+    cy.expect(remoteStorageNavigationPane.find(configurationsItem).exists());
+    cy.expect(remoteStorageNavigationPane.find(accessionTablesItem).exists());
```
Comment (nayimovag) [inline cypress/support/fragments/settings/remote-storage/remoteStorage.js:10]: if multiple assertions in a row, better to pass as an array

File: cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js
Comment (nayimovag) [inline cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js:1]: please rename file, maybe something like remote-storage-toast or remote-storage-callout

**Review by nayimovag — CHANGES_REQUESTED** [review]

File: cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js (hunk 1 above)
Comment (artsiomBandarenka) [reply cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js]: fixed

File: cypress/support/fragments/settings/remote-storage/configurations.js (hunk 2 above)
Comment (artsiomBandarenka) [reply cypress/support/fragments/settings/remote-storage/configurations.js]: fixed

File: cypress/support/fragments/settings/remote-storage/configurations.js (hunk 3 above)
Comment (artsiomBandarenka) [reply cypress/support/fragments/settings/remote-storage/configurations.js]: fixed

File: cypress/support/fragments/settings/remote-storage/remoteStorage.js (hunk 4 above)
Comment (artsiomBandarenka) [reply cypress/support/fragments/settings/remote-storage/remoteStorage.js]: fixed

File: cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js
Comment (artsiomBandarenka) [reply cypress/e2e/settings/remote-storage/remote-storage-configuration.cy.js:1]: done

File: cypress/support/fragments/settingsMenu.js [hunk 5]
```diff
@@ -103,4 +105,8 @@ export default {
   organizationCategoryPath: 'settings/organizations/category',
   // Tags
   tagsGeneralPath: 'settings/tags/general',
+
+  openRemoteStorageSettings: () => {
+    cy.do(Link({ href: including('settings/remote-storage') }).click());
+  },
```
Comment (nayimovag) [inline cypress/support/fragments/settingsMenu.js:111]: you dont need this method

File: cypress/e2e/settings/remote-storage/remote-storage-callout.cy.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Configurations } from '../../../support/fragments/settings/remote-storage';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
```
Comment (nayimovag) [inline cypress/e2e/settings/remote-storage/remote-storage-callout.cy.js:6]: 'TopMenu' is defined but never used. Allowed unused vars must match /React/u

Comment (artsiomBandarenka) [reply cypress/e2e/settings/remote-storage/remote-storage-callout.cy.js]: removed

**Review by nayimovag — APPROVED** [review]

**Pull request merged**